src/components/SiteDetail.js

```diff
@@ -72,63 +73,46 @@ const SiteDetail = ({ selectSite, projectFishFamilies }) => {
       </Typography>
     </Box>
   );
-
-  const siteChartCards = availableProtocolsForChart.map(({ name, property, title, type }) => {
-    const bleachingProtocol = name === 'bleachingqc';
-    const transectName = bleachingProtocol ? property : name;
-    const loadedSiteProtocol = currentSelectedSite && currentSelectedSite.protocols[transectName];
-    const bleachingSubItems =
-      currentSelectedSite && currentSelectedSite.protocols['colonies_bleached'];
-
-    const dataPolicy = loadedSiteProtocol && currentSelectedSite[`data_policy_${name}`];
-    const setToPrivate = dataPolicy === 'private';
-
-    const generatePrivateLabel = protocolTitle =>
-      `This data is unavailable because ${protocolTitle} Sample Units are set to Private for this project.`;
-
-    const convertContent = content => {
-      let contentArr = [];
-      for (const item in content) {
-        const newObject = {};
-        newObject[item] = content[item];
-        contentArr.push(newObject);
+  const siteChartCards = useMemo(() => {
+    const selectedSiteProtocols = currentSelectedSite?.protocols;
```

Comment:
is selectedSiteProtocols an array? or object? Is this a weird api thing?

Reply:
its an object